bpbuilder/builder

```diff
@@ -74,8 +74,11 @@ else
     cp /etc/deis-slugbuilder.yaml /etc/${SLUG_NAME}.yaml
 fi
 
-if [ -z "$S3EP" ]; then
-    S3EP=${HOST}:3000
+# if minio is in the cluster, use it. otherwise use fetcher
+if [[ -n "$MINIO_SERVICE_HOST" && -n "$MINIO_SERVICE_PORT" ]]; then
+  S3EP=${MINIO_SERVICE_HOST}:${MINIO_SERVICE_PORT}
+elif [ -z "$S3EP" ]; then
+  S3EP=${HOST}:3000
 fi
```

Comment:
@smothiki still trying to wrap my head around the moving pieces in builder, so thanks for these. does this script already push the tar to s3? I'll add the logic to push to minio there.